src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -117,12 +117,16 @@ public void shootAtPlayer() {
     public void ezShot(){//still needs code to stop from shooting same tile twice. Consider using code in else statement in hardShot()
```

Comment:
Just to clarify, does your function prevent the computer from firing at the same space yet?